src/js/components/reviewData/CertificationModal/CertifyButtons.jsx

```diff
@@ -5,33 +5,48 @@
 
 import React from 'react';
 import PropTypes from 'prop-types';
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
 
 const propTypes = {
     clickedCertifyButton: PropTypes.func,
     clickedCertifyCheckbox: PropTypes.func,
     closeModal: PropTypes.func,
     session: PropTypes.object,
     certified: PropTypes.bool,
-    loading: PropTypes.bool
+    loading: PropTypes.bool,
+    type: PropTypes.oneOf(['publish', 'certify', 'both'])
 };
 
 const defaultProps = {
     clickedCertifyButton: null,
     clickedCertifyCheckbox: null,
     closeModal: null,
     session: {},
-    certified: false
+    certified: false,
```

Comment:
I want this renamed given it applies to publishing too but honestly I can't think of any.